crates/ark/src/fixtures/dummy_frontend.rs

```diff
@@ -144,3 +182,54 @@ impl DerefMut for DummyArkFrontendNotebook {
         DerefMut::deref_mut(&mut self.inner)
     }
 }
+
+impl DummyArkFrontendRprofile {
+    /// Lock a frontend that supports `.Rprofile`s.
+    ///
+    /// NOTE: This variant can only be called exactly once per process,
+    /// because you can only load an `.Rprofile` one time.
+    ///
+    /// NOTE: Only one `DummyArkFrontend` variant should call `lock()` within
+    /// a given process.
```

Comment:
Do these two notes need to be merged?

Reply:
IIUC, we could potentially have one global Rprofile per test process, but we still could not have multiple test blocks because we need to exhaust all emitted messages within the first block?